src/components/Identification/index.js

```diff
@@ -24,17 +24,18 @@ import WarningIcon from '@material-ui/icons/Warning';
 import CloseIcon from '@material-ui/icons/Close';
 import Snackbar from '@material-ui/core/Snackbar';
 import IconButton from '@material-ui/core/IconButton';
-import './index.css';
-import GordonLoader from '../../../../components/Loader';
 import defaultGordonImage from './defaultGordonImage';
+import GordonLoader from '../Loader/index';
+import { windowBreakWidths } from '../../theme';
+import './index.css';
 
 export const Identification = props => {
   const CROP_DIM = 200; // pixels
   const [isImagePublic, setIsImagePublic] = useState(null);
   const [defaultUserImage, setDefaultUserImage] = useState(null);
   const [preferredUserImage, setPreferredUserImage] = useState(null);
-  const [hasPreferredImage, seHasPreferredImage] = useState(true);
-  const [switchPhotos, setSwitchPhotos] = useState(false);
+  const [hasPreferredImage, setHasPreferredImage] = useState(true);
+  const [isPhotosSwitched, setisPhotosSwitched] = useState(false);
   const [showCropper, setShowCropper] = useState(null);
```

Comment:
Fixed a typo mistake and variable name for a clearer understanding of the variable and it's functionality